documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/README.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: docs
+title: On-Page Actions
+permalink: /documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/
+---
+# On-Page Actions
+
+The **On-Page Actions** category contains test actions that simulate user actions on a tested web page.
+
+* ![Action icon](../../../../../images/actions/action-click-icon.svg) [Click](#click)
+* ![Action icon](../../../../../images/actions/action-double-click-icon.svg) [Double Click](#double-click)
+* ![Action icon](../../../../../images/actions/action-right-click-icon.svg) [Right Click](#right-click)
+* ![Action icon](../../../../../images/actions/action-type-text-icon.svg) [Type Text](#type-text)
+* ![Action icon](../../../../../images/actions/action-press-key-icon.svg) [Press Key](#press-key)
+* ![Action icon](../../../../../images/actions/action-drag-icon.svg) [Drag](#drag)
+* ![Action icon](../../../../../images/actions/action-drag-to-element-icon.svg) [Drag To Element](#drag-to-element)
+* ![Action icon](../../../../../images/actions/action-select-text-icon.svg) [Select Text](#select-text)
+* ![Action icon](../../../../../images/actions/action-select-editable-content-icon.svg) [Select Editable Content](#select-editable-content)
+* ![Action icon](../../../../../images/actions/action-select-text-content-icon.svg) [Select Text Area Content](#select-text-area-content)
+* ![Action icon](../../../../../images/actions/action-wait-icon.svg) [Pause](#pause)
+* ![Action icon](../../../../../images/actions/action-hover-icon.svg) [Hover](#hover)
+
+You can add these actions in two ways:
+
+* Add them to a test and specify its parameters manually.
+* You can record the major of on-page actions. When you interact with a tested web page, TestCafe Studio automatically adds corresponding on-page actions with their necessary parameters to a test.
```

Comment:
The two list items does not have consistent first sentences (which is not good).

```
* Add them to a test and specify its parameters manually.
* Record them using the Visual Test Recorder. You can record most of the on-page action. When you interact with a tested web page, TestCafe Studio automatically adds corresponding on-page actions with their necessary parameters to a test.
```